scipy/special/tests/test_support_alternative_backends.py

```diff
@@ -48,8 +48,6 @@ def test_rel_entr_generic(dtype):
 
 @pytest.mark.fail_slow(5)
 @array_api_compatible
-# @pytest.mark.skip_xp_backends('numpy', reason='skip while debugging')
-# @pytest.mark.usefixtures("skip_xp_backends")
```

Comment:
IIRC @mdhaber left this in deliberately as an instant reminder of the syntax (which has changed a few times) if one has to return to debugging here at a later point. But probably fine to remove if you feel strongly.

Reply:
I don't feel strongly, but it felt nice to clean up